testing/test_junitxml.py

```diff
@@ -619,6 +620,20 @@ def test_func(char):
     node.assert_attr(name="test_func[#x00]")
 
 
+def test_double_colon_split_issue469(testdir):
```

Comment:
Could you also add an integration test when using a test method? Just to be safe. :grin: